README.md

```diff
@@ -2,4 +2,38 @@
 
 AII is the automated installation infrastructure.
 
-More documentation will follow here.
+If you are serving host profiles and kickstart files with Apache httpd, access to can be restricted with the a config based on the following:
+
+```apacheconf
+# Use Aliases or set DocumentRoot
+Alias /profiles /var/quattor/web/htdocs/profiles
+Alias /kickstart /var/quattor/web/htdocs/kickstart
+
+<location /profiles>
+  Require all granted
+</location>
+
+HostnameLookups Double
+
+RewriteEngine On
+# The following ACL map can be used to specify hosts which can access all profiles
+RewriteMap ACLmap txt:/var/quattor/web/ACLmap.txt
+
+# Redirect every profile request to HTTPS
+RewriteCond %{HTTPS} !=on
+RewriteRule  ^/profiles/(.*)$ https://%{HTTP_HOST}/profiles/$1 [L]
+
+# Serve only the profile corresponding to the remote host if the host is not part of the ACLmap
+RewriteCond ${ACLmap:%{REMOTE_HOST}|NO} NO
```

Comment:
if you only need this for the aii server, and it is the same one as the one serving the profiles (like in this example); you can remove this, and use `cdburl = dir:///var/quattor/....` in aii-shellfe.conf

Reply:
Good point, will document this too.